assets/wizards/popups/views/segmentation/SingleSegment.js

```diff
@@ -66,6 +66,8 @@ const SingleSegment = ( { segmentId, setSegments, wizardApiFetch } ) => {
 	const isDirty =
 		segmentInitially !== null &&
 		JSON.stringify( segmentInitially ) !== JSON.stringify( segmentConfig );
+	const isEmpty = JSON.stringify( segmentConfig ) === JSON.stringify( DEFAULT_CONFIG );
+	console.log( segmentConfig, isEmpty );
```

Comment:
Stray `console.log`?

Reply:
Still seeing this `console.log`, but as soon as you remove it this is good to go!

Reply:
Done now 👍